### ISSUE TITLE
inconsistent results with esqlc (embedded SQL C) variable markers

### ISSUE THREAD

**Issue by clach04** [uncrustify/uncrustify#525]
The following esqlc file:

```
EXEC SQL BEGIN DECLARE SECTION;
    static char    *tbuf;
EXEC SQL END DECLARE SECTION;

void myfunc1()
{
    exec sql execute immediate :tbuf;
}

void myfunc2()
{
    EXEC SQL EXECUTE IMMEDIATE :tbuf;
}
```

when ran through uncrustify runs without error but the results are inconsistent, e.g. get:

```
exec sql execute immediate: tbuf;
...
EXEC SQL EXECUTE IMMEDIATE :tbuf;
```

So the workaround is to upper case the code ;-)

Looks like this is caused by tokenize_cleanup.cpp:

```
if ((chunk_is_str(pc, "EXEC", 4) && chunk_is_str(next, "SQL", 3)) ||
```

which appears to be a case sensitive compare?

**Comment by gmaurel** [comment]
Thanks for the message.

**Comment by gmaurel** [comment]
Fixed with #879
